daml/Marketplace/Registry.daml

```diff
@@ -36,6 +45,16 @@ template RegisteredIssuer
     key (operator, issuer) : (Party, Party)
     maintainer key._2
 
+    controller issuer can
+      RegisteredIssuer_UpdateProfile : ContractId RegisteredIssuer
+        with
+          newName : Text
+          newLocation : Text
+          newTitle : Text
+          newIssuerID : Text
```

Comment:
what if this party only wants to update their name? I assume then the rest of these "new" fields will actually equal the current profile configuration

Reply:
yep, the form fills in with the existing values and if you change the name field and submit, it resubmits the rest with the same original value